config/sync/workflows.workflow.nics_editorial_workflow.yml

```diff
@@ -4,7 +4,6 @@ status: true
 dependencies:
```

Comment:
This file doesn't seem relevant for breadcrumb or menu purposes. Accidental commit?